### DISCUSSION TITLE
Syncing from my desktop client does not work

### DISCUSSION THREAD

**Discussion by plrnr** [nextcloud/all-in-one#781] [Questions] [answered]
hi,

Client versions I tried: 3.5.1 and 3.3.6

Added some pictures into nextcloud/photos folder and the status shows 'pending sync' and stays like that. Can I get some help on resolving this?
Mobile app sync works just fine.

**Comment by szaimen** [comment, marked as answer]
I guess you should probably report this to the desktop client repo if this should still happen...
